### PR TITLE
Airbag optimization

### PR DESCRIPTION
**Pull request by roguic**

### CODE REVIEW

File: engine/source/airbag/get_volume_area.F90 [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+!Copyright>        As an alternative to this open-source version, Altair also offers Altair Radioss
+!Copyright>        software under a commercial license.  Contact Altair to discuss further if the
+!Copyright>        commercial version may interest you: https://www.altair.com/radioss/.
+      module get_volume_area_mod
+      contains
+! ======================================================================================================================
+!                                                   procedures
+! ======================================================================================================================
+!! \brief here is a small description of the routine, [after the header]
+!! \details if needed, more details can be added here
```
Comment (sebastienVilleneuve) [inline engine/source/airbag/get_volume_area.F90:29]: @roguic 

can you add brief description of the routine ( even if the routine name is already giving this information)

File: engine/source/airbag/init_global_monvol_frontier.F90 [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+!Copyright>
+!Copyright>        As an alternative to this open-source version, Altair also offers Altair Radioss
+!Copyright>        software under a commercial license.  Contact Altair to discuss further if the
+!Copyright>        commercial version may interest you: https://www.altair.com/radioss/.
+      module init_global_frontier_monvol_mod
+      contains
+! ======================================================================================================================
+!                                                   procedures
+! ======================================================================================================================
+!! \brief here is a small description of the routine, [after the header]
```
Comment (sebastienVilleneuve) [inline engine/source/airbag/init_global_monvol_frontier.F90:28]: same here , can you add description ?

File: engine/source/airbag/init_monvol_omp_structure.F90 [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+!Copyright>        As an alternative to this open-source version, Altair also offers Altair Radioss
+!Copyright>        software under a commercial license.  Contact Altair to discuss further if the
+!Copyright>        commercial version may interest you: https://www.altair.com/radioss/.
+      module init_monvol_omp_structure_mod
+      contains
+! ======================================================================================================================
+!                                                   procedures
+! ======================================================================================================================
+!! \brief here is a small description of the routine, [after the header]
+!! \details if needed, more details can be added here
```
Comment (sebastienVilleneuve) [inline engine/source/airbag/init_monvol_omp_structure.F90:29]: same here , can you add description ?

**Review by sebastienVilleneuve — CHANGES_REQUESTED** [review]
@roguic  , some minor changes ( new routine brief description)

**Review by sebastienVilleneuve — APPROVED** [review]

**Review by laurent-altr — APPROVED** [review]

**Pull request merged**